code/modules/mob/living/silicon/ai/death.dm

```diff
@@ -31,6 +31,8 @@
 			SSshuttle.emergency.mode = SHUTTLE_DOCKED
 			SSshuttle.emergency.timer = world.time
 			priority_announce("Hostile enviroment resolved. You have 3 minutes to board the Emergency Shuttle.", null, 'sound/AI/shuttledock.ogg', "Priority")
+		for(var/obj/item/weapon/pinpointer/point in world)

```

Comment:
Make a global pinpointer list and have the pinpointer add itself to that on New? That's the only thing I can think of.